tests/braille-specs/en-GB-g2.yaml

```diff
@@ -333,128 +333,128 @@ tests:
   - [first, ⠋⠌]
   - [it, ⠭]
   - [its, ⠭⠎]
-  - ["it's", ⠭⠄⠎, xfail: only backward translation fails]
+  - ["it's", ⠭⠄⠎, xfail: {backward: "only backward translation fails"}]
   - [Llandaff, ⠠⠇⠇⠁⠝⠙⠁⠋⠋, xfail: both forward and backward translation fails]
```

Comment:
I guess this comment doesn't make much sense anymore

Reply:
I thought it was fine to keep it, for clarity.

Reply:
OK, fine